packages/gasket-plugin-elastic-apm/lib/middleware.js

```diff
@@ -20,6 +19,13 @@ const apm = require('elastic-apm-node');
  * @param {Response}  res     The server response
  */
 async function customizeTransaction(gasket, req, res) {
+  const apm = gasket.apm;
+
+  if (!apm.isStarted()) {
+    gasket.logger.warning('Elastic APM has not been started properly.');
```

Comment:
Is this log necessary or important? Seems like it would be very noisy for local development if it logs for each request.